tests/results/DiffTest.py

```diff
@@ -17,6 +17,11 @@ def test_add_lines(self):
         self.uut.add_lines(0, ['t'])
         self.uut.add_lines(0, [])
 
+    def test_add_line(self):
```

Comment:
You'll have put that under assert statement, other wise you see you're getting conflicterror. 
Dosomething like
`self.assertRaises(ConflictError, self.uut.add_line, 0, 't')`
after adding the first line.